StorageUtility/Windows/StorageUtility.cpp

```diff
@@ -203,6 +203,30 @@ eErrorCode GetStorageAdapterProperty( HANDLE Handle, sStorageAdapterProperty& Ad
     return( eErrorCode::None );
 }
 
+eErrorCode GetPhysicalDiskNumber(HANDLE Handle, U32& PhysicalDiskNumber)
```

Comment:
I think we need only Device Path distinguishing drives. Device path is unique for a Drive